src/sdk/tracer.ts

```diff
@@ -13,6 +13,10 @@ import * as AgentsModule from "langchain/agents";
 import * as ToolsModule from "langchain/tools";
 import * as chromadb from "chromadb";
 
+// @ts-expect-error Azure SDK does not provide valid declarations for their CommonJS build
+// https://github.com/Azure/azure-sdk-for-js/issues/28877
+import * as azureOpenAI from "@azure/openai";
```

Comment:
This line of code was raising a compiler error because the Azure OpenAI SDK, at least at `v1.0.0-beta.12` does not provide valid TypeScript declarations for their CommonJS build.